malasakit-django/pcari/templates/base.html

```diff
@@ -36,6 +37,18 @@
         <h1 id="main-heading">{% block main-heading %}Malasakit{% endblock %}</h1>
       </div>
     </header>
+    <page-summary>
+      <div class="container">
+        <center>
```

Comment:
`<center>` is [deprecated](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/center) (use `text-align: center` in the CSS).